### ISSUE TITLE
Exporting incorrectly

### ISSUE THREAD

**Issue by ericwooley** [gpac/mp4box.js#52]
since https://github.com/gpac/mp4box.js/commit/eaa50adb4526936db751e5e57ad68842f59370fd
the export is like this

~~~
if (typeof exports !== 'undefined') {
	exports.MP4Box = MP4Box;	
}
~~~

I noticed some old libs were depending on old versions, so I tried updating and found that none of them imported correctly.

It might be too late, because you don't want to make a breaking change but I think it should be exported like this:
~~~
if (typeof exports !== 'undefined') {
	exports = MP4Box;	
}
~~~
Unless you are now exporting multiple things.

**Comment by cconcolato** [comment]
Thanks for reporting. The current export scheme is a bit ugly. But, I'd like to export multiple things: MP4Box and ISOFile. Just did not have the time to change the code and test. Is it breaking anything for you? I could try to do it differently.



> I noticed some old libs were depending on old versions, so I tried updating and found that none of them imported correctly.

I didn't get that part.

**Comment by ericwooley** [comment]
I noticed some old libs were depending on old versions, so I tried updating and found that none of them imported correctly.

Sorry, let me clarify that a bit. I think i should hire an editor...

I stole code from https://github.com/jhiesey/videostream and tried to use it with the latest version of mp4boxjs, but it imports mp4boxjs like this `var MP4Box = require('mp4box');`, which appearently with older versions of mp4boxjs. So I was a bit confused as to why I kept getting the error that mp4boxjs was not a function when I tried to instantiate it. It took me a minute to figure out I needed to require like this 
`var MP4Box = require('mp4box').MP4Box;`

Kind of a moot point if you plan on exporting multiple things, since the way you are doing it is definitely correct for that case.
